hepcrawl/spiders/elsevier_spider.py

```diff
@@ -554,20 +537,28 @@ def _get_ref_years(self, ref):
     def _parse_references(self, ref, label):
         """Parse all the references."""
         reference = {}
-        textref = ref.xpath(".//ce:textref//text()").extract()
+
+        def add_items_to_reference(**kwargs):
```

Comment:
Well, I'm using this only in two spiders. A lot of the items in a reference require more than simple if checks, so using a generic function doesn't work. At least I couldn't think anything better; feel free to advice me on that matter.